src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -1,27 +1,41 @@
 import styles from './MovieDetails.module.css';
 import React from "react";
 import {MovieDetailsData} from "../../types/MovieDetailsData";
+import placeholderImage from "../../assets/images/placeholderPoster.jpg";
+import formatRuntime from "../../utils/formatRuntime";
 
 interface MovieDetailsProps {
     movieDetails: MovieDetailsData;
+    handleBackToSearch: () => void;
 }
 
-const MovieDetails: React.FC<MovieDetailsProps> = ({movieDetails}) => {
-    const {imageUrl, title, releaseDate, genres, runtime, rating, description} = movieDetails;
+const MovieDetails: React.FC<MovieDetailsProps> = ({movieDetails, handleBackToSearch}) => {
+    const {poster_path, title, release_date, genres, runtime, vote_average, overview} = movieDetails;
     return (
-        <div className={styles.movieDetails}>
-            <img src={imageUrl} alt={title}/>
+        <div className={styles.movieDetails} data-testid="movie-details">
+            <img className={styles.movieDetailsImage}
+                 src={!poster_path ? placeholderImage : poster_path}
+                 alt={title}
+                 onError={(e) => {
+                     e.currentTarget.src = placeholderImage;
+                     e.currentTarget.onerror = null;
+                 }
+                }
+            />
```

Comment:
`onError={(e) => {
                     e.currentTarget.src = placeholderImage;
                     e.currentTarget.onerror = null;
                 }`
                 
                 Your code looks very dirty. Always try to move away handlers to function, and it this JSX provide only name. It will be easier to read
                 `onError={handleError}`